src/components/Auth/Auth.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import Tabs from "@material-ui/core/Tabs";
+import Tab from "@material-ui/core/Tab";
+import SignIn from "./SignIn";
+import SignUp from "./SignUp";
+import { withStyles } from "@material-ui/styles";
+
+const styles = {
+  container: {
+    maxWidth: "18rem",
+    margin: "0 auto"
+  },
+  tab: {
+    minWidth: "50%"
+  }
+};
+
+class Auth extends Component {
+  state = {
```

Comment:
Use PropTypes like this: https://github.com/ihor4568/spotty-react/blob/master/src/components/shared/Main.js

static propTypes = {
  classes: PropTypes.object.isRequired
};

Reply:
fixed